alembic/versions/4033221ea50c_rename_jobtriggermodel.py

```diff
@@ -0,0 +1,58 @@
+"""Renamed JobTriggerModel in ProjectEventModel
+
+Revision ID: 4033221ea50c
+Revises: b58f55c0112c
+Create Date: 2023-05-24 08:16:53.631139
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "4033221ea50c"
+down_revision = "b58f55c0112c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("ALTER TYPE jobtriggertype RENAME TO projecteventtype")
+    op.rename_table("job_triggers", "project_events")
+    op.alter_column("project_events", "trigger_id", new_column_name="event_id")
+
+    op.execute("ALTER SEQUENCE job_triggers_id_seq RENAME TO project_events_id_seq")
+    op.execute("ALTER INDEX job_triggers_pkey RENAME TO project_events_pkey")
+    op.execute(
+        "ALTER INDEX ix_job_triggers_trigger_id RENAME TO ix_project_events_event_id"
+    )
+
+    op.alter_column("pipelines", "job_trigger_id", new_column_name="project_event_id")
+    op.drop_constraint("runs_job_trigger_id_fkey", "pipelines", type_="foreignkey")
```

Comment:
Why we are dropping the constraint?

Reply:
Because I didn't find a way to rename it. So the only solution for me it was to drop it and re-create it with the new name.